dpxdt/client/capture.js

```diff
@@ -77,6 +76,12 @@ if (config.cookies) {
     });
 }
 
+// Add username and password as a parameter for HTTP basic auth
+if (config.httpUserName && config.httpPassword) {
```

Comment:
Same comment (I have below) about allowing httpUserName to be specified but httpPassword to be empty. May be worth splitting this into two if statements and assignments.